tests/conftest.py

```diff
@@ -37,12 +39,17 @@
 from sqlalchemy.ext.asyncio import AsyncEngine, AsyncSession
 from starlette.types import ASGIApp
 
+for name, logger in logging.root.manager.loggerDict.items():
+    if name.startswith("phoenix.") and isinstance(logger, logging.Logger):
+        logger.handlers.clear()
+        logger.addHandler(logging.StreamHandler())
+        logger.setLevel(logging.DEBUG)
```

Comment:
Looks like this is leftover from debugging.

Reply:
since these are tests, maybe they should have it too

Reply:
Does this show up in the output if you run `pytest` in verbose mode?

Reply:
i'm not sure, i have to try it (i never did)